### ISSUE TITLE
ArrayIndexOutOfBoundsException in CacheListAdapter

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#9457]
From Google Play for 2020.11.23

```
java.lang.ArrayIndexOutOfBoundsException: 
  at cgeo.geocaching.ui.CacheListAdapter.getPositionForSection (CacheListAdapter.java:771)
  at android.widget.FastScroller.getPosFromItemCount (FastScroller.java:1302)
  at android.widget.FastScroller.onItemCountChanged (FastScroller.java:534)
  at android.widget.AbsListView.onLayout (AbsListView.java:2208)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.widget.LinearLayout.setChildFrame (LinearLayout.java:1829)
  at android.widget.LinearLayout.layoutVertical (LinearLayout.java:1673)
  at android.widget.LinearLayout.onLayout (LinearLayout.java:1582)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at androidx.appcompat.widget.ActionBarOverlayLayout.onLayout (ActionBarOverlayLayout.java:530)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.widget.LinearLayout.setChildFrame (LinearLayout.java:1829)
  at android.widget.LinearLayout.layoutVertical (LinearLayout.java:1673)
  at android.widget.LinearLayout.onLayout (LinearLayout.java:1582)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at com.android.internal.policy.DecorView.onLayout (DecorView.java:799)
  at android.view.View.layout (View.java:21973)
  at android.view.ViewGroup.layout (ViewGroup.java:6263)
  at android.view.ViewRootImpl.performLayout (ViewRootImpl.java:3108)
  at android.view.ViewRootImpl.performTraversals (ViewRootImpl.java:2618)
  at android.view.ViewRootImpl.doTraversal (ViewRootImpl.java:1749)
  at android.view.ViewRootImpl$TraversalRunnable.run (ViewRootImpl.java:7692)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:966)
  at android.view.Choreographer.doCallbacks (Choreographer.java:790)
  at android.view.Choreographer.doFrame (Choreographer.java:725)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:951)
  at android.os.Handler.handleCallback (Handler.java:883)
  at android.os.Handler.dispatchMessage (Handler.java:100)
  at android.os.Looper.loop (Looper.java:241)
  at android.app.ActivityThread.main (ActivityThread.java:7617)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:492)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:941)
```

**Comment by eddiemuc** [comment]
Duplicate of #9048, should be fixed with PR #9428

**Comment by Lineflyer** [comment]
Thanks @eddiemuc I overlooked that issue.
